### DISCUSSION TITLE
leafmap.foliumap not showing layers

### DISCUSSION THREAD

**Discussion by jllucero** [opengeos/leafmap#45] [Q&A] [answered]
Hello. Leafmap.foliumap is not showing layers on the final map.  I'm trying to add two layers to the basemap.  The first layer is a geojson file.  And the second layer is a kml file.  I'm using "add_geojson" and "add_kml" to make the layers. But when I display the map, only the basemap shows.  I'd appreciate some guidance.
Thank you.

**Comment by giswqs** [comment, marked as answer]
Did you look at the notebook example? https://leafmap.org/notebooks/10_add_vector. It works without any problem.

**Reply by jllucero** [reply]
I found my problem.  I was adding a json layer as if it was geojson first and then added a kml layer after.  So nothing was showing.  But after I added the json as a dataframe using "add_gdf" both layers show up.  And I saved it  to html and it works fine in the browser.  Thank you again!  Leafmap is awesome!
